portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -52,11 +56,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get comments from the form
     String name = getParameter(request, /*name=*/"name", /*defaultValue=*/"");
     String comment = getParameter(request, /*name=*/"comment", /*defaultValue=*/"No comments");
+    long timestamp = System.currentTimeMillis();
     // TODO: validate request parameters
 
     // Modify the state of server.
     addComment(name, comment);
 
+    Entity commentEntity = new Entity("Comment");
```

Comment:
Maybe move these data persistent logic into addComment?

Reply:
Fixed

Reply:
I will just leave this here in this PR since i changed in another PR haha